app/src/containers/Newevent/Newevent.js

```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import DateTimeField from 'react-bootstrap-datetimepicker';
 import * as showActions from '../../redux/newEventReducers';
-import messages from '../../helper/messages';
+import messages from '../../helper';
```

Comment:
Look at 3 row in app/src/containers/About/About.js. Why they are different? Check everywhere.